### PR TITLE
zaki/enable_btns_mt5

### PR DESCRIPTION
**Pull request by zaki-hanafiah**
Enabled buttons on mt5_type_of_accounts page.

### CODE REVIEW

File: src/javascript/common/champion.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ const Champion = (function() {
             'metatrader-5'         : { module: MT5 },
             'champion-trader'      : { module: ClientType },
             'economic-calendar'    : { module: ClientType },
-            'mt5-types-of-accounts': { module: MT5 },
```
Comment (negarn) [inline src/javascript/common/champion.js]: why was this emptied?

Comment (zaki-hanafiah) [reply src/javascript/common/champion.js]: sorry, my bad. updating..

**Review by negarn — APPROVED** [review]

**Comment by negarn** [comment]
remember to remove client to fix the failing test for https://travis-ci.org/ChampionFX/champion-static/builds/274440366#L463

File: src/javascript/common/champion.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ const Champion = (function() {
             'metatrader-5'         : { module: MT5 },
             'champion-trader'      : { module: ClientType },
             'economic-calendar'    : { module: ClientType },
-            'mt5-types-of-accounts': {},
+            'mt5-types-of-accounts': { module: MT5 },
```
Comment (kelly-binary) [inline src/javascript/common/champion.js]: i think it is safe to remove this line since it is now a static page?

Comment (negarn) [reply src/javascript/common/champion.js]: @kellybinary but it is calling .tabs on its onload function (https://github.com/ChampionFX/champion-static/pull/268/files#diff-f6da2a8419af33e18bf32125df3b4c1aL9) so we need it, right?

**Review by mohammad-deriv — APPROVED** [review]

**Review by kelly-binary — APPROVED** [review]

**Pull request merged**